static/js/api.js

```diff
@@ -37,50 +37,74 @@ function AtcRestClient (endpoint) {
       }
     });
   };
+
+  this.primary_call = function (method, urn, callback, data) {
+    this.api_call(primary, method, urn, callback, data);
+  };
+
+  this.secondary_call = function (method, urn, callback, data) {
+    this.api_call(secondary, method, urn, callback, data);
```

Comment:
I need to do some cleanup on this file anyway. I'll probably rename all these methods.